torchvision/models/regnet.py

```diff
@@ -587,6 +587,21 @@ class RegNet_Y_16GF_Weights(WeightsEnum):
             "acc@5": 98.054,
         },
     )
+    IMAGENET1K_SWAG_LINEAR_V1 = Weights(
+        # This model has trunk weight from weakly supervised learning described in https://arxiv.org/pdf/2201.08371.pdf
+        # with only linear head finetuned to IMAGENET1K dataset
+        # This model is suitable for users that want to fine tune the pretrained trunk on other downstream datasets
```

Comment:
FYI: I had a discussion with @NicolasHug around this last week. We need a way to document enum values because some of the new enum values are not necessarily self explanatory.